test.py

```diff
@@ -0,0 +1 @@
+print("hello f#cking repository");
```

Comment:
wow, print? :D test.py?
that's something new, but we probably need just a couple lines in a readme.md file